errors/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## To be Released
 
+## v2.3.0
+
 * feat: add `Is` and `As` to match standard library
 * fix: remove `NoteMask`. `Notef` should be use instead.
```

Comment:
Don't we consider that this item should lead to a new major version?

Reply:
Hmm indeed I didn't check I was breaking something here.

Actually I would want to make a v3 with the removal of errgo, it will come.